src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -264,25 +270,52 @@
                       (do ((:dismiss props)) ((:reload-after-delete props))) ;if success, update the table?
                       (swap! state assoc :server-error (get-parsed-response false))))})))})
 
+(react/defc- ClusterErrorViewer
```

Comment:
Previously this was embedded in `NotebooksTable`, but I made it a full-fledged component because it needs to manage some state -- it needs to fetch the cluster details from Leo before displaying the errors.